airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -21,7 +21,7 @@
 import json
 import warnings
 from functools import cached_property
-from typing import TYPE_CHECKING, Any, Dict, List, cast
+from typing import TYPE_CHECKING, Any, Dict, List, Sequence, cast
```

Comment:
we should move these import in block `if TYPE_CHECKING:`

Reply:
We are using it for more than type-checking. That's why we need it out of `if TYPE_CHECKING:`

Reply:
Nope, It doesn't for it.